telegram-history-dump.rb

```diff
@@ -129,9 +135,13 @@ def dump_dialog(dialog)
       end
 
       if dump_msg
-        process_media(dialog, msg)
-        if $dumper.dump_msg(dialog, msg) == false
-          keep_dumping = false
+        begin
+          process_media(dialog, msg)
+          if $dumper.dump_msg(dialog, msg) == false
+            keep_dumping = false
+          end
+        rescue Timeout::Error
+          keep_dumping = true
```

Comment:
Yes, maybe you are right. As said, this is the first time I write in Ruby. If you are completely sure I'll remove the check, but I think a double check can never hurt.